e2e/suites/application/page-titles.test.ts

```diff
@@ -124,5 +132,47 @@ describe('Page titles', () => {
                     expect(browser.getTitle()).toContain(label);
                 });
         });
+
     });
-});
+
+    describe('on viewer mode', () => {
+        beforeAll( async (done) => {
+           const resp = (await nodesApi.createFile(file));
+            fileId = resp.data.entry.id
```

Comment:
instead of these 2 lines you can just write
`fileId = (await nodesApi.createFile(file)).entry.id;`